tests/attendee/attendee-repository.php

```diff
@@ -206,6 +221,46 @@ public function test_get_events_for_user() {
 		$this->assertEmpty( $this->repository->get_events_for_user( $another_user + 1 ) );
 	}
 
+	public function test_is_new_translation_contributor() {
+		$this->set_normal_user_as_current();
+		$now      = new DateTimeImmutable( 'now', new DateTimeZone( 'UTC' ) );
+		$user1_id = 52;
+		$user2_id = 53;
+		$user3_id = 54;
+
+		// create 10 translations  for $user2_id before event start.
+		for ( $i = 0; $i < 10; $i++ ) {
+			$this->translation_factory->create( $user2_id );
+
+		}
+		// create 11 translations  for $user3_id before event start.
+		for ( $i = 0; $i < 11; $i++ ) {
+			$this->translation_factory->create( $user3_id );
+		}
+
+		$event1_id  = $this->event_factory->create_active( array(), $now->modify( '+1 day' ) );
+		$attendee11 = new Attendee( $event1_id, $user1_id );
+		$attendee12 = new Attendee( $event1_id, $user2_id );
+		$attendee13 = new Attendee( $event1_id, $user3_id );
+		$this->repository->insert_attendee( $attendee11 );
+		$this->repository->insert_attendee( $attendee12 );
+		$this->repository->insert_attendee( $attendee13 );
+		$event = $this->event_repository->get_event( $event1_id );
+
+		$contributor_ids     = array( $attendee11->user_id(), $attendee12->user_id(), $attendee13->user_id() );
+		$new_contributor_ids = array();
+		$translations_counts = $this->translation_repository->count_translations_before( $contributor_ids, $event->start() );
```

Comment:
I know that this is how new contributors (after the performance improvement in #225) is calculated but we should test this inherently.

Thus, can we add a `is_new_contributor()` implementation with this test that uses `count_translations_before` internally?